helpers/report_hunter.py

```diff
@@ -0,0 +1,109 @@
+"""
+track down the latest version of all reports
+generate an index HTML page with links to all reports
+"""
+
+
+from dataclasses import dataclass
+from os.path import join
+from pathlib import Path
+
+import jinja2
+from google.cloud import storage
+
+from cpg_utils import to_path
+
+
+JINJA_TEMPLATE_DIR = Path(__file__).absolute().parent / 'templates'
+TEMPLATE = 'gs://cpg-{dataset}-main-web/reanalysis'
+WEB_TEMPLATE = 'https://main-web.populationgenomics.org.au'
+
+
+cohorts = [
```

Comment:
If we can pull this list from the master config when you have it that would be grand.